articles/_posts/2015-09-08-release-v4.0.0.md

```diff
@@ -0,0 +1,290 @@
+* **util**: The `util.is*()` functions have been deprecated, beginning with deprecation warnings in the documentation for this release, users are encouraged to seek more robust alternatives in the npm registry, (Sakthipriyan Vairamani) [#2447](https://github.com/nodejs/node/pull/2447).
+* **v8**: Upgrade to version 4.5.103.30 from 4.4.63.30 (Ali Ijaz Sheikh) [#2632](https://github.com/nodejs/node/pull/2632).
+ - Implement new `TypedArray` prototype methods: `copyWithin()`, `every()`, `fill()`, `filter()`, `find()`, `findIndex()`, `forEach()`, `indexOf()`, `join()`, `lastIndexOf()`, `map()`, `reduce()`, `reduceRight()`, `reverse()`, `slice()`, `some()`, `sort()`. See https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/TypedArray for further information.
+ - Implement new `TypedArray.from()` and `TypedArray.of()` functions. See https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/TypedArray for further information.
+ - Implement arrow functions, see https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions for further information.
+ - Full ChangeLog available at https://github.com/v8/v8-git-mirror/blob/4.5.103/ChangeLog
+-->
+* **child_process**: `ChildProcess.prototype.send()`와 `process.send()`가 모든 플랫폼에서 비동기로 동작하므로 메시지를 보낸 후 호출될 선택적인 콜백 파라미터가 추가되었습니다. 예시: `.send(message[, sendHandle][, callback])` (Ben Noordhuis) [#2620](https://github.com/nodejs/node/pull/2620).
+* **node**: "io.js" 코드를 "Node.js"로 변경했습니다. (cjihrig) [#2367](https://github.com/nodejs/node/pull/2367).
+* **node-gyp**: 이번 릴리즈에는 갱신된 버전의 node-gyp가 포함되었습니다. 이번 node-gyp는 Node.js와 io.js의 모든 버전과 나이틀리와 릴리스 후보 빌드 모두에서 동작합니다. io.js v3와 Node.js v4부터 애드온을 빌드할 때 전체 소스를 다운로드 하지 않고 headers tarball만 다운로드 받습니다. (Rod Vagg) [#2700](https://github.com/nodejs/node/pull/2700)
+* **npm**: 2.13.3에서 2.14.2 버전으로 업그레이드했습니다. (Kat Marchán) [#2696](https://github.com/nodejs/node/pull/2696).
+* **timers**: 0.12 구현을 포팅하고 일부 버그를 수정해서 timer의 성능이 개선되었습니다. (Jeremiah Senkpiel) [#2540](https://github.com/nodejs/node/pull/2540), (Julien Gilli) [nodejs/node-v0.x-archive#8751](https://github.com/nodejs/node-v0.x-archive/pull/8751) [nodejs/node-v0.x-archive#8905](https://github.com/nodejs/node-v0.x-archive/pull/8905)
+* **util**: `util.is*()` 함수를 폐기해서 이번 릴리스부터는 문서에 폐기 경고를 표시하기 시작했습니다. 사용자는 npm 저장소에서 더 신뢰할 수 있는 대체제를 찾기를 권장합니다. (Sakthipriyan Vairamani) [#2447](https://github.com/nodejs/node/pull/2447).
+* **v8**: 4.4.63.30에서 4.5.103.30 버전으로 업그레이드했습니다. (Ali Ijaz Sheikh) [#2632](https://github.com/nodejs/node/pull/2632).
+ - 새로운 `TypedArray` 프로토타입 메서드를 구현했습니다: `copyWithin()`, `every()`, `fill()`, `filter()`, `find()`, `findIndex()`, `forEach()`, `indexOf()`, `join()`, `lastIndexOf()`, `map()`, `reduce()`, `reduceRight()`, `reverse()`, `slice()`, `some()`, `sort()`. <https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/TypedArray for further information>를 참고하세요.
```

Comment:
https\://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/TypedArray for further information → https\://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/TypedArray